pytorch_tools/torchfold_test.py

```diff
@@ -37,6 +38,7 @@ def logits(self, enc, embed):
 class TorchFoldTest(unittest.TestCase):
 
     def test_rnn(self):
+        return
```

Comment:
?
if you just want to run one test case, you can always set it from command line:
`python any_test.py TestCaseName.test_function_name`

Reply:
I was too lazy figuring out how to run individual test cases in PyCharm and accidentally leaked this.